contracts/Batcher.sol

```diff
@@ -1,5 +1,7 @@
 pragma solidity 0.7.5;
 
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
```

Comment:
Do we want to use this in WalletSimple for batching functions as well? Was one of the messages I got from slither